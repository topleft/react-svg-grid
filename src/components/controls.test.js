import React from 'react';
import Controls from './controls.jsx';
import TestRenderer from 'react-test-renderer';

it('renders correctly', () => {
    const inputValues = {};
    const tree = TestRenderer
        .create(<Controls inputValues="{inputValues}">Facebook</Controls>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});