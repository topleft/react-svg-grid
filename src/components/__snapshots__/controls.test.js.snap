// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="controlsContainer"
  style={
    Object {
      "backgroundColor": "#FF0058",
      "textAlign": "center",
    }
  }
>
  <div
    style={
      Object {
        "marginBottom": "15px",
      }
    }
  >
    <label
      htmlFor="itemsPerRow"
      style={
        Object {
          "color": "#CCCBB6",
          "display": "block",
          "fontSize": "12px",
          "position": "bottom",
          "transform": "rotate(-7deg)",
        }
      }
    >
      PER ROW
    </label>
    <input
      min=""
      name="itemsPerRow"
      onChange={[Function]}
      style={
        Object {
          "backgroundColor": "rgba(255,255,255, 0.2)",
          "margin": "10px auto",
          "textAlign": "center",
          "transform": "rotate(-7deg)",
          "width": "25%",
        }
      }
      type="number"
      value={undefined}
    />
  </div>
  <div
    style={
      Object {
        "marginBottom": "15px",
      }
    }
  >
    <label
      htmlFor="itemsPerRow"
      style={
        Object {
          "color": "#CCCBB6",
          "display": "block",
          "fontSize": "12px",
          "position": "bottom",
          "transform": "rotate(-7deg)",
        }
      }
    >
      X AXIS
    </label>
    <input
      min=""
      name="horizontalMargin"
      onChange={[Function]}
      style={
        Object {
          "backgroundColor": "rgba(255,255,255, 0.2)",
          "margin": "10px auto",
          "textAlign": "center",
          "transform": "rotate(-7deg)",
          "width": "25%",
        }
      }
      type="number"
      value={undefined}
    />
  </div>
  <div
    style={
      Object {
        "marginBottom": "15px",
      }
    }
  >
    <label
      htmlFor="itemsPerRow"
      style={
        Object {
          "color": "#CCCBB6",
          "display": "block",
          "fontSize": "12px",
          "position": "bottom",
          "transform": "rotate(-7deg)",
        }
      }
    >
      Y AXIS
    </label>
    <input
      min=""
      name="verticalMargin"
      onChange={[Function]}
      style={
        Object {
          "backgroundColor": "rgba(255,255,255, 0.2)",
          "margin": "10px auto",
          "textAlign": "center",
          "transform": "rotate(-7deg)",
          "width": "25%",
        }
      }
      type="number"
      value={undefined}
    />
  </div>
  <div
    style={
      Object {
        "marginBottom": "15px",
      }
    }
  >
    <label
      htmlFor="itemsPerRow"
      style={
        Object {
          "color": "#CCCBB6",
          "display": "block",
          "fontSize": "12px",
          "position": "bottom",
          "transform": "rotate(-7deg)",
        }
      }
    >
      RADIUS
    </label>
    <input
      min={1}
      name="radius"
      onChange={[Function]}
      style={
        Object {
          "backgroundColor": "rgba(255,255,255, 0.2)",
          "margin": "10px auto",
          "textAlign": "center",
          "transform": "rotate(-7deg)",
          "width": "25%",
        }
      }
      type="number"
      value={undefined}
    />
  </div>
</div>
`;
